bindings/wasm/identity_wasm/src/storage/wasm_storage_signer.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020-2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+use core::convert::TryFrom;
```

Comment:
```suggestion
use core::convert::TryFrom;
```

`core` is automatically brought in scope, there shouldn't be a need for this.